app/services/weather_service.rb

```diff
@@ -13,17 +11,6 @@ def get_forecast(lat, lon)
    JSON.parse(response.body, symbolize_names: true)
   end
 
-  def trip_forecast(lat, lon)
-  response = conn.get("data/2.5/onecall") do |req|
-    req.params[:lat] = lat
-    req.params[:lon] = lon
-    req.params[:exclude] = "minutely,current,daily"
-    req.params[:units] = "imperial"
-    req.params[:appid] = ENV["OPEN_WEATHER_API_KEY"]
-   end
-   JSON.parse(response.body, symbolize_names: true)
-  end
```

Comment:
Was using this for RoadTrip but realized that it was probably unnecessary and I could just as easily parse the data from the first method.